orders/src/main/java/works/weave/socks/orders/controllers/OrdersController.java

```diff
@@ -114,12 +144,18 @@ CustomerOrder newOrder(@RequestBody NewOrderResource item) {
 //        return ResponseEntity.ok(resources);
 //    }
 
-    private String parseId(String href) {
-        Pattern idPattern = Pattern.compile("[\\w-]+$");
-        Matcher matcher = idPattern.matcher(href);
-        if (!matcher.find()) {
-            throw new IllegalStateException("Could not parse user ID from: " + href);
+    private float calculateTotal(List<Item> items) {
+        float amount = 0F;
+        float shipping = 4.99F;
```

Comment:
The shipping amount is hard coded in both the front end and back end at the moment(?) we might trip over this later